src/lib.rs

```diff
@@ -113,9 +113,9 @@ extern crate term_rewriting;
 pub mod domains;
 mod ec;
 mod gp;
-pub mod hmtrs;
 pub mod lambda;
 pub mod pcfg;
+pub mod trs;
```

Comment:
Ah, yes - I was thinking the crate needed a sleeker moniker. `trs` is a definite improvement.